tests/test_dbos.py

```diff
@@ -1508,6 +1509,12 @@ def blocked_workflow() -> None:
                 blocked_workflow()
         assert assert_current_dbos_context().workflow_deadline_epoch_ms is None
         handle = DBOS.start_workflow(blocked_workflow)
+        status = handle.get_status()
+        assert status.workflow_timeout_ms == 100
+        assert (
+            status.workflow_deadline_epoch_ms is not None
+            and status.workflow_deadline_epoch_ms > time.time() * 1000
```

Comment:
What is this test? I think we want to make sure the deadline is set to a reasonable number related to the actual timeout ms.